docs/internals/contributing/triaging-tickets.txt

```diff
@@ -411,58 +411,3 @@ the ticket database:
 .. _`creating an account on Trac`: https://www.djangoproject.com/accounts/register/
 .. _password reset page: https://www.djangoproject.com/accounts/password/reset/
 .. _Django Forum: https://forum.djangoproject.com/
-
-Bisecting a regression
```

Comment:
As the author of this section, I don't think moving it is appropriate. Bisecting is an important part of triaging new tickets. This information is out of place in internals/git.txt.

Also, I am fairly sure I've linked to this section elsewhere on the web, so it would break those links. Moving and renaming sections shouldn't be done lightly. Thanks.

Reply:
I agree with Tim. Triaging tickets is not the first thing that newcomers will/should do. Also, `docs/internals/git.txt` describes the structure of the Django repository, not tips on using Git.

Reply:
@timgraham @felixxm Ok, I also agree, so I've reverted this particular change.